### PR TITLE
0.4.3 release

### PR DESCRIPTION
**Pull request by mediaminister**
Preparing for the v0.4.3 release:
- updated README
- updated addon.xml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,17 @@ Please report any issues or bug reports on the [GitHub Issues](https://github.co
...
+## v0.4.3 (2019-09-xx)
+- French translation (@brunoduc)
+- Various translations fixes (@vlmaksime, @dagwieers)
+- Ensure Kodi 19 compatibility (@mediaminister)
+- Configurable temporary download directory for devices with limited space (@horstle)
+- Configurable Widevine CDM update frequency (@horstle)
+- Fix add-on settings crashes when InputStream Adaptive is missing (@mediaminister)
+- Improve unicode character support (@mediaminister)
+- Improve testing framework (@dagwieers)
+- Improve code structure (@dagwieers)
```
Comment (dagwieers) [inline README.md]: As I mentioned before, I am not sure if these two belong in the changelog.
It is very generic, and something we will be doing as part of changes anyway.
So I hate to see this being part of the changelog in every new release (see previous notes).

Comment (mediaminister) [reply README.md]: Removed.

**Comment by mediaminister** [comment]
@dagwieers I plan to release this around 21:00 (Brussels time)
Please let me know beforehand if you still want something changed to the release.

**Comment by dagwieers** [comment]
@mediaminister No, I think this is fine to be released as-is.

**Comment by dagwieers** [comment]
And I think we have been doing a pretty good job too ;-)

**Comment by emilsvennesson** [comment]
You definitely have! 👏 I'm home soon and will hopefully finish the updated Swedish translation before you release!

@mediaminister I also think it's time for you to add yourself to the authors in addon.xml. :-)

**Comment by mediaminister** [comment]
@emilsvennesson I'll wait for your commit with swedish translations.

**Comment by dagwieers** [comment]
@mediaminister Congratulations with the promotion ;-)

**Comment by mediaminister** [comment]
Thanks. :-)

**Pull request merged**